### PR TITLE
[MSG] Fix inbox header title when there are no messages

### PR DESCRIPTION
**Pull request by mennenia**
Fixes [this issue](https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=2&projectKey=IN&selectedIssue=IN-7)

The zerostate inbox test is actually outdated also. I'll sort that when I come back. 

#skip_new_tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Inbox/Conversations/__tests__/index-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "#skip_new_tests".
  </td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: src/lib/Components/Inbox/Conversations/__tests__/Conversations-tests.tsx [hunk 1]
```diff
@@ -10,6 +10,11 @@ it("looks correct when rendered", () => {
   expect(tree).toMatchSnapshot()
 })
 
+it("doesn't render the header when there are no messages", () => {
+  const tree = renderer.create(<Conversations me={mePropsEmpty} relay={{ hasMore: jest.fn() } as any} />).toJSON()
+  expect(tree).toMatchSnapshot()
+})
```
Comment (orta) [inline src/lib/Components/Inbox/Conversations/__tests__/Conversations-tests.tsx]: TBH, this test doesn't _actually_ test that, it just verifies the JSON - would recommend adding a check for the lack of that specifically - you can see some examples here: https://github.com/artsy/emission/blob/master/src/lib/Components/Consignments/Screens/__tests__/Confirmation-tests.tsx#L87-L108

Comment (mennenia) [reply src/lib/Components/Inbox/Conversations/__tests__/Conversations-tests.tsx]: ah ok, will push after dinner 👌

**Comment by mennenia** [comment]
@orta that what you had in mind?

**Comment by orta** [comment]
Yep 👍

**Pull request merged**